lib/maze.rb

```diff
@@ -7,7 +7,7 @@
 # Glues the various parts of MazeRunner together that need to be accessed globally,
 # providing an alternative to the proliferation of global variables or singletons.
 module Maze
-  VERSION = '9.20.0'
+  VERSION = '9.21.0'
```

Comment:
Should be a patch version bump for a fix.
```suggestion
  VERSION = '9.20.1'
```